animation/geometry.h

```diff
@@ -337,20 +337,30 @@ namespace animation
                     (const_cast <Box const &> (*this))->apply_visitor (visitor);
                 }
 
+                T const & topLeft () const
+                {
+                    return tl;
+                }
+
+                T const & bottomRight () const
+                {
+                    return br;
+                }
+
             private:
 
-                T tr;
-                T bl;
+                T tl;
+                T br;
         };
 
         template <typename T>
         template <typename U>
         bool Box <T>::contains (U const &p) const
         {
-            auto const x1 = dimension::get <0> (tr);
-            auto const x2 = dimension::get <0> (bl);
-            auto const y1 = dimension::get <1> (tr);
-            auto const y2 = dimension::get <1> (bl);
+            auto const x1 = dimension::get <0> (tl);
+            auto const x2 = dimension::get <0> (br);
+            auto const y1 = dimension::get <1> (tl);
+            auto const y2 = dimension::get <1> (br);
```

Comment:
If you're changing the corners used for extracting the limits, and you're not changing the comparison, then it means the comparison is wrong now or before. Seems like it was wrong before. Or was it just the name of the vars that was incorrect? If so, then maybe making more emphasis on that fact in the commit message would be good?

Reply:
In this case, the values themselves didn't actually change, only the names, which were wrong. I'll re-emphasize that in the commit message.

Reply:
Commit message fixed.